node/clients/checkout.ts

```diff
@@ -46,17 +46,25 @@ export class Checkout extends JanusClient {
   public addItem = (
     orderFormId: string,
     items: Array<Omit<OrderFormItemInput, 'uniqueId' | 'index' | 'options'>>,
-    salesChannel?: string,
-    allowOutdatedData?: string[]
-  ) =>
+    options?: {
+      salesChannel?: string,
+      allowOutdatedData?: string[],
+      splitItem?: boolean 
+    }) =>
```

Comment:
Hmm, I don't think we can do this, as it's a breaking change. Maybe it's better to keep `salesChannel` in the same place. Is that possible?
```suggestion
    salesChannel?: string,
    options?: {
      allowOutdatedData?: string[],
      splitItem?: boolean 
    }) =>
```